cypress/integration/page.spec.js

```diff
@@ -291,7 +320,7 @@
         button().should('have.text', 'Choose a user');
       });
 
-      it('should not show users list by default', () => {
+      it.skip('should not show users list by default', () => {
```

Comment:
This test is skipped. Ensure that it is intentionally skipped and consider enabling it if it is relevant to the test suite.